nixos/modules/services/web-servers/vencloud.nix

```
@@ -0,0 +1,100 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.vencloud;
```

```suggestion
  cfg = config.services.vencloud;
  
  settingsFormat = pkgs.formats.keyValue { };
```

It'd probably be better to use a standard format here, as suggested in [RFC 42](https://github.com/NixOS/rfcs/blob/25c3f524631000b851375e7b96223a56e71cc0e2/rfcs/0042-config-option.md)